docs/useCheckForApplicationUpdate.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: default
+title: "useCheckForApplicationUpdate"
+---
+
+# `useCheckForApplicationUpdate`
+
+This hook regularly checks whether the application source code has changed and calls the provided function when an update is available. To detect updates, it fetches the current URL at regular intervals and compares a hash of the response content.
```

Comment:
Shouldn't we explain what hash we are talking about ?

Reply:
> a hash of the response content

It's already there, no?

Reply:
Or simply

```suggestion
This hook regularly checks whether the application source code has changed and calls the provided function when an update is available. To detect updates, it fetches the current URL at regular intervals and compares the hash of the response content.
```

Reply:
Something like 
> To detect updates, it fetches the current URL at regular intervals, hash the response content (eg; the HTML) and compares it to the current content hash